acac_portal/Pipfile

```diff
@@ -0,0 +1,30 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "==2.2.13"
```

Comment:
Is this version compatible with the DRF version?
Maybe you guys can shift to a newer version of Django

Reply:
Yes, it is compatible with DRF but yeah we can shift to Django 3 I believe

Reply:
Shift to a newer version and push the changes

Reply:
ok